nservicebus/azure-storage-persistence/index.md

```diff
@@ -18,9 +18,9 @@ Certain features of NServiceBus require persistence to permanently store data. A
 
 ## How to enable persistence with Azure Storage Services
 
-First add a reference to the assembly that contains the Azure storage persisters. The recommended way of doing this is by adding a NuGet package reference to the `NServiceBus.Azure` package.
+First add a reference to the assembly that contains the Azure storage persisters. When working with NServiceBus Version 5 or lower the recommended way of doing this is by adding a NuGet package reference to `NServiceBus.Azure`. For NServiceBus Version 6 and above the NuGet package reference will be to `NServiceBus.Persistence.AzureStorage`.
```

Comment:
We mention the recommended way but no alternative. Are there alternatives? Can we be more assertive?

> In NServiceBus Version 5 and below, add a NuGet package reference to `NServiceBus.Azure`.
> In NServiceBus Version 6 and above, add a NuGet package reference to `NServiceBus.Persistence.AzureStorage`.